tripper/backends/ontopy.py

```diff
@@ -51,73 +51,86 @@ def __init__(
     def triples(self, triple: "Triple") -> "Generator":
         """Returns a generator over matching triples."""
 
-        def to_literal(o, d):
-            """Returns a literal from (o, d)."""
-            if isinstance(d, str) and d.startswith("@"):
-                lang, datatype = d[1:], None
+        def to_literal(object_, datum):
+            """Returns a literal from (object_, datum)."""
+            if isinstance(datum, str) and datum.startswith("@"):
+                lang, datatype = datum[1:], None
             else:
-                lang, datatype = None, d
-            return Literal(o, lang=lang, datatype=datatype)
+                lang, datatype = None, datum
+            return Literal(object_, lang=lang, datatype=datatype)
 
-        s, p, o = triple
+        subject, predicate, object_ = triple
         abb = (
-            None if s is None else self.onto._abbreviate(s),
-            None if p is None else self.onto._abbreviate(p),
-            None if o is None else self.onto._abbreviate(o),
+            None if (subject) is None else self.onto._abbreviate(subject),
+            None if (predicate) is None else self.onto._abbreviate(predicate),
+            None if (object_) is None else self.onto._abbreviate(object_),
         )
-        for s, p, o in self.onto._get_obj_triples_spo_spo(*abb):
+        for subject, predicate, object_ in self.onto._get_obj_triples_spo_spo(*abb):
```

Comment:
Good practice is to keep the line length less than 79 characters, since longer lines are difficult to read on a standard 80 character wide terminal.

Reply:
"Standard" is _very_ subjective in this context.
Most modern coding environments and terminals support larger/more variable terminal windows.
Furthermore, this project is not implementing a terminal-specific API (like a command-line interface), which is less of a reason to specifically support terminals.

I instead fully support the more modern take on line lengths for more easy readability in modern code editors. Something that has been investigated and determined to be a line-length of 88 characters, which is a compromise line-length.